ow_lander/scripts/constants.py

```diff
@@ -60,7 +60,7 @@
 PAN_MIN        = -3.2
```

Comment:
`from math import pi` can be removed from this script

Reply:
Good catch, done.